features/suzie_views_a_petition.feature

```diff
@@ -94,3 +94,15 @@ Feature: Suzie views a petition
     When I view the petition
     Then I should see "Parliament debated this petition on 26 October 2015"
     And I should see "Waiting for 1 day for Parliament to publish the debate outcome"
+
+  Scenario: Suzie does not see information about future signature targets when viewing a closed petition
+    Given a petition "Spend more money on Defence" has been closed
+    When I view the petition
+    Then I should not see "At 10,000 signatures..."
+    Then I should not see "At 100,000 signatures..."
+
```

Comment:
I'd be tempted to add some more scenarios here for when a closed petition has more than 10,000 signatures and more than 100,000 that you don't see those blocks but you do see the response and debate information. Those seem like possible failure scenarios if someone messes up the view logic in future.